carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carla_to_carma_twist

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+"""
+Subscribe from CARLA :nav_msgs::Odometry
+    Topic: /carla/{}/odometry
+
+Publish to CARMA :geometry_msgs::TwistStamped
+    Topic: /hardware_interface/vehicle/twist
+"""
+from os import stat
+import rospy
+from geometry_msgs.msg import TwistStamped
+from nav_msgs.msg import Odometry
+from cav_msgs.msg import GuidanceState
+
+twist_pub = rospy.Publisher('/hardware_interface/vehicle/twist', TwistStamped, queue_size=1)
+gs_state = 0
+
+def guidance_state_callback(data):
+    """
+    callback guidance state
+    """
+    global gs_state
+    gs_state = data.state
+
+def odometry_callback(data):
+    """
+    callback odometry
+    """
+    if gs_state != GuidanceState.ENGAGED:
```

Comment:
Thanks for the information. The reason why I added this condition is because, when the CARMA guidance state is ENGAGED and CARLA virtual vehicle is following the vehicle command from CARMA to move then if I suddenly reset the CARMA platform (not relaunch that, just click the circle button in CARMA-Web-UI and select restart), the guidance state will turn back to initial state but the topic of `hardware_interface/vehicle_cmd` is still publishing the message which might only start to publish the message when the guidance state is ENGAGED. I was wondering this situation is because I send twist always but after I test it again, the topic of `hardware_interface/vehicle_cmd` keeping to publish the message even guidance state not ENGAGED seems like not related with what I thought. So will remove the part of checking guidance state in this script.